src/scripts/main.js

```diff
@@ -1,3 +1,62 @@
+/* eslint-disable prefer-promise-reject-errors */
+/* eslint-disable promise/param-names */
+/* eslint-disable no-unused-vars */
 'use strict';
 
-// write your code here
+function showNotification(message, type) {
+  const notification = document.createElement('div');
+
+  notification.className = type;
+  notification.setAttribute('data-qa', 'notification');
+  notification.textContent = message;
+  document.body.appendChild(notification);
+}
+
+const promise1 = new Promise((resolve) => {
+  const logo = document.querySelector('.logo');
+
+  logo.addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+});
+
+const promise2 = new Promise((_, reject) => {
```

Comment:
The parameter name '_' is used to ignore the resolve function in the promise. While this is valid, it might be clearer to use a more descriptive name or comment to indicate why it's being ignored.